devel/libxkbcommon/Portfile

```diff
@@ -44,15 +47,15 @@ configure.args      -Ddefault_library=both \
                     -Denable-x11=false
 
 if {${subport} eq ${name}} {
-    revision        0
+    revision        1
 
     depends_lib-append \
                     port:libxml2 \
                     port:xkeyboard-config
 }
 
 subport ${name}-x11 {
-    revision        0
+    revision        1
```

Comment:
Did you see the `@rpath` problem with libxkbcommon-x11 too? I didn't, but I don't understand why not.

Reply:
I didn't, but decided to revbump it anyway just to be sure all systems were fixed. Perhaps not needed, but I didn't know for sure.